internal/k8s/status_test.go

```diff
@@ -47,7 +47,7 @@ func TestSetIngressRouteStatus(t *testing.T) {
 			t.Helper()
 			var gotPatchBytes []byte
 			s := runtime.NewScheme()
-			ingressroutev1.AddKnownTypes(s)
+			ingressroutev1.AddToScheme(s) // nolint:errcheck
```

Comment:
I think we should just add the error checking here as well.

Reply:
Fixed.